pkg/containerd/utility_windows.go

```diff
@@ -0,0 +1,15 @@
+// +build windows
+
+package containerd
+
+import (
+	util2 "github.com/rancher/k3s/pkg/util"
+)
+
+func OverlaySupported(root string) error {
+	return util2.ErrUnsupportedPlatform
+}
+
+func FuseoverlayfsSupported(root string) error {
+	return util2.ErrUnsupportedPlatform
```

Comment:
Wrap the error to provide some context on what in particular isn't supported
```suggestion
	util2 "github.com/rancher/k3s/pkg/util"
	"github.com/pkg/errors"
)

func OverlaySupported(root string) error {
	return errors.Wrap(util2.ErrUnsupportedPlatform, "overlayfs is not supported")
}

func FuseoverlayfsSupported(root string) error {
	return errors.Wrap(util2.ErrUnsupportedPlatform, "fuse-overlayfs is not supported")
```